ros2launch/ros2launch/api/api.py

```diff
@@ -16,15 +16,16 @@
 
 from collections import OrderedDict
 import os
-from typing import List
-from typing import Text
-from typing import Tuple
+import pathlib
+from tempfile import TemporaryDirectory
+from typing import List, Text, Tuple, Union
```

Comment:
They obviously prefer having these on their own lines. Unless you're getting linter errors, changing these is just noise.

Reply:
Black changed it and I was comitting the pathlib block, fixing